sdk-actors/src/main/java/io/dapr/actors/client/ActorClient.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ */
+
+package io.dapr.actors.client;
+
+import io.dapr.client.DaprApiProtocol;
+import io.dapr.client.DaprHttpBuilder;
+import io.dapr.config.Properties;
+import io.dapr.v1.DaprGrpc;
+import io.grpc.Channel;
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+import reactor.core.publisher.Mono;
+
+/**
+ * Holds a client for Dapr sidecar communication. ActorClient should be reused.
+ */
+public class ActorClient implements AutoCloseable {
```

Comment:
Have this implement `DaprClient` as well?

Reply:
I had this in a version but it would expose the invokeActorMethod publicly while it is supposed to be internal do the ActorProxy's package.

Reply:
oh ok .